docs/contributing/core/index.md

```diff
@@ -50,7 +50,7 @@ However, the following links and sections below may be helpful.
 
 Installing Python is beyond the scope of this documentation.
 However, it is recommended to use a Python version manager, {term}`pyenv` that allows you to install multiple versions of Python on your development environment without destroying your system's Python.
-Plone requires Python version {SUPPORTED_PYTHON_VERSIONS}.
+Plone requires Python version {SUPPORTED_PYTHON_VERSIONS_PLONE60}.
```

Comment:
We should use the Plone 6.1 versions here.
```suggestion
Plone requires Python version {SUPPORTED_PYTHON_VERSIONS_PLONE61}.
```

Reply:
See previous comment.